OJS.Workers.SubmissionProcessors/ExecutionTypeFilters/LocalSubmissionsFilteringService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace OJS.Workers.SubmissionProcessors.ExecutionTypeFilters
+{
+    using System.Collections.Generic;
+    using OJS.Workers.Common.Models;
+
+    using static OJS.Workers.Common.ExecutionStrategiesConstants.NameMappings;
+
+    public class LocalSubmissionsFilteringService
+        : SubmissionFilteringServiceBase
+    {
+        protected override ISet<ExecutionStrategyType> EnabledExecutionStrategyTypes
+            => new HashSet<ExecutionStrategyType>();
```

Comment:
I don't get it how this HashSet is empty and in the same time the worker has enabled strategies?

Reply:
It does not. It supports all, except the unsupported

Reply:
Looking at this check, if the `EnabledExecutionStrategyTypes` is empty, the result will be `false`, thus the submission will not be eligible for execution. What am I missing?

![image](https://user-images.githubusercontent.com/15523019/120330722-c1da0900-c2f5-11eb-82ec-c9592c84a703.png)